Sources/SwiftParser/TokenPrecedence.swift

```diff
@@ -192,47 +192,266 @@ enum TokenPrecedence: Comparable {
     case  // Literals
     .Self, .false, .nil, .`self`, .super, .true:
       self = .identifierLike
+
     // MARK: Expr keyword
     case  // Keywords
-    .as, .is, .try,
+    .as, .is, .some, .try,
+      .await, .reasync, .async,
       // We don't know much about which contextual keyword it is, be conservative an allow considering it as unexpected.
       // Keywords in function types (we should be allowed to skip them inside parenthesis)
       .rethrows, .throws,
       // Consider 'any' a prefix operator to a type and a type is expression-like.
       .Any,
       // 'where' can only occur in the signature of declarations. Consider the signature expression-like.
       .where,
+      .each,
       // 'in' occurs in closure input/output definitions and for loops. Consider both constructs expression-like.
       .in:
       self = .exprKeyword
+
     case  // Control-flow constructs
     .defer, .do, .for, .guard, .if, .repeat, .switch, .while,
       // Secondary parts of control-flow constructs
       .case, .catch, .default, .else,
       // Return-like statements
       .break, .continue, .fallthrough, .return, .throw, .then, .yield:
       self = .stmtKeyword
+
     // MARK: Decl keywords
     case  // Types
-    .associatedtype, .class, .enum, .extension, .protocol, .struct, .typealias,
+    .associatedtype, .class, .enum, .extension, .protocol, .struct, .typealias, .actor, .Sendable,
       // Access modifiers
       .fileprivate, .internal, .private, .public, .static,
       // Functions
       .deinit, .func, .`init`, .subscript,
       // Variables
-      .let, .var,
+      .let, .var, .weak,
       // Operator stuff
       .operator, .precedencegroup,
       // Declaration Modifiers
       .__consuming, .final, .required, .optional, .lazy, .dynamic, .infix, .postfix, .prefix, .mutating, .nonmutating, .convenience, .override, .package, .open,
-      .__setter_access, .indirect, .nonisolated, .distributed, ._local,
-      .inout, ._mutating, ._borrowing, ._consuming,
+      .__setter_access, .indirect, .isolated, .nonisolated, .distributed, .unchecked, ._local,
+      .inout, ._mutating, ._borrow, ._borrowing, .borrowing, ._consuming, .consuming, .consume,
+      ._backDeploy, .backDeployed, .inline, ._specialize, .availability, .unavailable, .deprecated, ._spi,
       // Misc
       .import:
       self = .declKeyword
-    default:
-      // Treat all keywords that weren't handled above as expression keywords as a fallback option.
-      // FIXME: We should assign a token precedence to all keywords
+
```

Comment:
Just a general question. 
Are all keyword still used or are there leftovers from previous major Swift versions? 

If yes, can some of those be removed?